datasets/gov/mita/horizon.csv

```diff
@@ -0,0 +1,20 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/mita/horizon,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,Horizontas2020,,,,,,,,,Duomenys apie Lietuvos pareiškėjų finansuotas paraiškas mokslinių tyrimų ir inovacijų programoje „Horizontas 2020“,
+,,,,,Projekto_nr,string,,,,3,open,,Projekto ID,
+,,,,,Projekto_akronimas,string,,,,3,open,,Projekto akronimas,
+,,,,,Lietuvos_dalyvis_projekte,string,,,,3,open,,Lietuvos dalyvis projekte,
```

Comment:
Čia kaip suprantu turėtu būti ryšys su juridinių asmenų registru?

- [ ] https://github.com/atviriduomenys/manifest/issues/115

Reply:
Ar gerai suprantu, kad net kai neatitinka pilnai pavadinimai reikia nurodyti ryšį tarp rinkinių(Vienama rinkinyje lietuviškos ir mažosios, kitame be Lietuviškų didžiosios raidės)?

Reply:
Taip, jei pavadinimai neatitinka, vis tiek reikia nurodyti ryšį, nurodant atitinkamą brandos lygį:

- `level = 2` - jei pavadinimai neatitinka JAR.
- `level = 3` - jei pavadinimai atitinka JAR.
- `level = 4` - jei yra ne pavadinimai o registracijos numeris.

Nurodžius 2 brandos lygį, ryšys bus tik metaduomenyse, bet patys duomenys nebus jungiami, nes jie nesusijungia. Plačiau [dokumentacijoje](https://atviriduomenys.readthedocs.io/dsa/ref.html#brandos-lygis).

Dar vienas niuansas, kol kas 2 ir 3 brandos lygio rinkinių aprašyti galimybės nėra, t.y. nepraeis `spinta check` patikrinimo. Todėl kol kas tokias vietas tik žymiu komentaruose taip:

- [ ] https://github.com/atviriduomenys/spinta/issues/208

Ir dar vienas dalykas, Registrų centras, kol kas nėra parengęs JAR struktūros aprašo. Todėl vėl, analogiškai, tik pažymiu komentaruose, kad būtų galima vėliau prie to grįžti.